src/pages/checkout-page/mui-tel-input/mui-tel-input.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC, useEffect, useState } from "react";
+import { MuiTelInput } from "mui-tel-input";
+import { ICountry } from "../../../state/state";
+
+interface ITelInput {
+  styleSx: object;
```

Comment:
If this is style then maybe it would better to use CssProperties instead of object?